docs/components/breadcrumb/README.md

```diff
@@ -316,7 +316,7 @@ Use the ```truncationThreshold``` prop to alter when truncation occurs:
 
 The below image and example code shows using ```truncationThreshold``` to set truncation to occur at 50%.
```

Comment:
@mhewson didn't we delete this prop? how did this content end up back here?

Reply:
yep, we can just delete everything below "Use the truncationThreshold prop to alter when truncation occurs:" instead

Reply:
swell --